supervisor/resolution/const.py

```diff
@@ -39,11 +40,13 @@ class IssueType(str, Enum):
     MISSING_IMAGE = "missing_image"
     UPDATE_FAILED = "update_failed"
     UPDATE_ROLLBACK = "update_rollback"
+    FATAL_ERROR = "fatal_error"
 
 
 class SuggestionType(str, Enum):
     """Sugestion type."""
 
     CLEAR_FULL_SNAPSHOT = "clear_full_snapshot"
     CREATE_FULL_SNAPSHOT = "create_full_snapshot"
-    SYSTEM_REPAIR = "system_repair"
+    EXECUTE_UPDATE = "execute_update"
+    EXECUTE_REPAIR = "execute_repair"
```

Comment:
`SYSTEM_REPAIR` is used in the codebase, those instances need to be renamed to  `EXECUTE_REPAIR`